src/app/ApplicationTemplate.Presentation/Framework/Diagnostics/IDiagnosticsService.cs

```diff
@@ -20,6 +20,7 @@ public interface IDiagnosticsService
 	/// Throws an exception from the main thread.
 	/// </summary>
 	/// <param name="ct">The cancellation token.</param>
+	/// <returns><see cref="Task"/>.</returns>
```

Comment:
Not sure this brings value?

Reply:
Warns otherwise its simple and concise. Don't know if you want to do something about it, having return doc is not something we should remove from the analyzer or styles.